common/containers/Tabs/SendTransaction/components/NonceField.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Help from 'components/ui/Help';
+import { translateRaw } from 'translations';
+import { isPositiveInteger } from 'utils/helpers';
+
+interface PublicProps {
+  placeholder: string;
+  value: number | null | undefined;
+  onChange(value: number): void;
+}
+
+const isValidNonce = (value: string | null | undefined) => {
+  let valid;
```

Comment:
This could be a more clean way of expressing this
```js
const valid = value === '0' ? true 
                            : !value ? false : isPositiveInteger(parseInt(value, 10))```

Reply:
😁 I'm not sure if I like a nested ternary better, but I'll think about how to improve this.